miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -159,14 +174,32 @@ abstract class MiniAppMessageBridge {
 
     private fun onRequestPermission(callbackObj: CallbackObj) {
```

Comment:
As mentioned above, I don't think we need to change the JavaScript method name, so probably you can remove this function.

Reply:
I have updated the PR, now I am using this when `requestDevicePermission` has not been implemented.